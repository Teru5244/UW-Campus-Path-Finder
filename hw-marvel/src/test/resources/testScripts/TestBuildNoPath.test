# Test if correctly built the graph from noPath.csv

LoadGraph np noPath.csv

ListNodes np

ListChildren np n1
ListChildren np n2
ListChildren np n3