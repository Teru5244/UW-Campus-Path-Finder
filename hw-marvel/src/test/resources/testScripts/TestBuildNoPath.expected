# Test if correctly built the graph from noPath.csv

loaded graph np

np contains: n1 n2 n3

the children of n1 in np are:
the children of n2 in np are:
the children of n3 in np are: